CloudNotes/CloudNotes/Controllers/ContentViewController.swift

```diff
@@ -8,53 +8,173 @@ import UIKit
 
 class ContentViewController: UIViewController {
     // MARK: - Property
-    private var textView: UITextView?
+    private var contentTextView: UITextView = {
+        var textView = UITextView()
+        textView.backgroundColor = .lightGray
+        textView.returnKeyType = .done
+        if UITraitCollection.current.userInterfaceIdiom == .phone {
+            textView.font = .systemFont(ofSize: 20, weight: .bold)
+        } else {
+            textView.font = .systemFont(ofSize: 25, weight: .bold)
+        }
+        return textView
+    }()
     var memo: String?
+    var memoEntity: MemoEntity?
+    var originalMemoContent: String?
 
     // MARK: - Life Cycle
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        if let memo = memo {
+            contentTextView.text = memo
+        } else if let memoEntity = memoEntity {
+            contentTextView.text = memoEntity.content
+        }
+    }
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .lightGray
         configureNavigationBar()
-        applyAdaptiveLayoutByDevice(textView: configureTextView)
+        configureTextView()
+        scrollWhenContentTextViewDidAppear()
+        scrollWhenKeyboardWillAppear()
+        setMemoIfNewMemoOrOriginalMemo()
+
+        setMemoListTableViewDelegate()
+        contentTextView.delegate = self
```

Comment:
`contentTextView` 를 생성하는 곳에 코드를 작성하면 어떨까요?
https://github.com/yagom-academy/ios-cloud-notes/pull/67/files#diff-dfb649ba102719cb80ffd656ef03e07598e938b5a34d19864336b1e246196bbcR11

Reply:
오~! 해당 클로저 내부에서는 ContentViewController의 인스턴스를 self로 접근이 안되는 것으로 알고있고, self가 해당 타입의 인스턴스로 알고 있어서 인스턴스를 만들어서 넣어주려고 해봤더니 아래와 같은 경고 문구가 떴습니다.

> Instance will be immediately deallocated because property 'delegate' is 'weak'

혹시 어떤 방식으로 contentTextView의 delegate의 설정이 가능한지 키워드나 참고할만한 자료를 공유해주실 수 있으실까요?ㅠㅠ